mage_ai/frontend/components/PipelineDetail/AddNewBlocks/utils.ts

```diff
@@ -25,6 +25,7 @@ const getDataSourceTypes = (
     return {
       [BlockTypeEnum.DATA_LOADER]: [
         DataSourceTypeEnum.KAFKA,
+        DataSourceTypeEnum.RABBITMQ,
```

Comment:
could you help sort this list in alphabetical order?

Reply:
Fixed on new commit (57b943b2dfdcf4d1ab128bf663facbd64e20beee)